homeassistant/components/google_mail/services.yaml

```diff
@@ -0,0 +1,51 @@
+set_vacation:
+  name: Set Vacation
+  description: Set vacation responder settings for Google Mail.
+  target:
```

Comment:
My impression is the best practice here is to use the config entry id as the input (which newly has a selector) rather than a device then getting config entries out of it

Reply:
I would think config entry is to be used when there is no device or entity available to distinguish one entry from another.